### PR TITLE
adding synthetics optimization quickstart

### PR DESCRIPTION
**Pull request by shredtechular**
# Summary

Create a quickstart for optimizing New Relic Synthetics. This quickstart package contains a pre-built dashboard.

## Pre merge checklist

- [ X ] Did you check you NRQL syntax? - Does it work?
- [ X ] Did you include an InstallPlan and Documentation reference?
- [ X ] Are all documentation links publically accessible?
- [ X ] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [ X ] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [ X ] Does the PR contain a screenshot for each of your dashboards?
- [ X ] Do your screenshots show data?
- [ X ] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [ X ] Did you check that your alerts actually work?

### CODE REVIEW

**Review by josephgregoryii — COMMENTED** [review]
A couple of comments:

I believe there is a typo in the `png` for the dashboard - 
`synthetcis-optimization.png` -> `synthetics-optimization.png`

Also, the image has a very high vertical sizing. Would you mind cropping the dashboard to have a more standard desktop-width image for the screenshots? If we take a look in the preview, it will show an idea of what I mean.

**Comment by shredtechular** [comment]
Great catches, thank you!  I have fixed the image title typo and cropped the image to make it fit better and make it more readable in the preview area.

**Comment by brammerl** [comment]
Doing a demo session on Wednesday January 4, 2023 to finalize approval

**Comment by aswanson-nr** [comment]
Hey @shredtechular, I missed this during the demo yesterday, but you'll need an install plan to make this quickstart "installable" in NR1. It looks like we already have an install plan that should work, I'll add a suggestion to the PR!

File: quickstarts/synthetics-optimization/config.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+documentation:
+  - name: Synthetics Usage Queries
+    url: >-
+      https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts/#synthetics-usage
+    description: |
+      This document provides additional queries that can be run to determine Synthetics usage.
+
+# Content / Design
+icon: logo.png
+website: https://www.newrelic.com
```
Comment (aswanson-nr) [inline quickstarts/synthetics-optimization/config.yml:44]:
```suggestion
website: https://www.newrelic.com

installPlans:
  - third-party-synthetics-endpoint-availability
```

Comment (shredtechular) [reply quickstarts/synthetics-optimization/config.yml:44]: Good catch @aswanson-nr !  I wasn't sure about that option when creating the config, but that makes sense to me now!  Thanks for the suggestion and finding that!!

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**